data/styles/clear/include/Icons.mapcss

```diff
@@ -3010,50 +3010,64 @@ area|z15-[amenity=sanitary_dump_station]
 
 /* 7.1 Parking */
 
-node|z16[amenity=parking],
-area|z16[amenity=parking]
-{icon-image: parking-s.svg;icon-min-distance: 25;}
-node|z16[amenity=parking][access=permissive],
-node|z16[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
-node|z17[amenity=parking],
-area|z17[amenity=parking]
-{icon-image: parking-m.svg;}
-node|z17[amenity=parking][access=permissive],
-node|z17[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
+node|z16-[amenity=parking],
+area|z16-[amenity=parking]
+{icon-image: parking-m.svg;icon-min-distance: 5;}
 node|z18-[amenity=parking],
 area|z18-[amenity=parking]
-{icon-image: parking-m.svg;font-size: 11;}
+{font-size: 11;}
+
+node|z16-[amenity=parking][access=private],
+area|z16-[amenity=parking][access=private],
+node|z16-[amenity=parking][access=no],
+area|z16-[amenity=parking][access=no],
+node|z16-[amenity=parking][parking=underground],
+area|z16-[amenity=parking][parking=underground],
+node|z16-[amenity=parking][location=underground],
+area|z16-[amenity=parking][location=underground]
+{icon-image: ;}
+
+node|z16-[amenity=parking][fee?],
+area|z16-[amenity=parking][fee?]
+{icon-image: parking_pay-m.svg;icon-min-distance: 5;}
+node|z18-[amenity=parking][fee?],
+area|z18-[amenity=parking][fee?]
+{icon-image: parking_pay-m.svg;font-size: 11;}
+
+node|z18-[amenity=parking][access=private],
+area|z18-[amenity=parking][access=private],
+node|z18-[amenity=parking][access=no],
+area|z18-[amenity=parking][access=no]
+{icon-image:parking_private-m.svg;font-size: 11;}
+
+node|z16-[amenity=parking][parking=street_side],
+area|z16-[amenity=parking][parking=street_side],
+node|z16-[amenity=parking][parking=lane],
+area|z16-[amenity=parking][parking=lane]
+{icon-image: ;icon-min-distance: 1;}
```

Comment:
Will not setting a min-distance force it to 0? I thought values have to be overwritten otherwise it'll inherit the min-distance from the base `[amenity=parking]`? Similar to `icon-image: ;`

Reply:
Yeap, you're right!